cmd/components.go

```diff
@@ -23,21 +24,32 @@ var ComponentsCmd = &cobra.Command{
 	Short: "List all Dapr components. Supported platforms: Kubernetes",
 	Run: func(cmd *cobra.Command, args []string) {
 		if kubernetesMode {
-			err := kubernetes.PrintComponents(componentsName, componentsOutputFormat)
+			if allNamespaces {
+				resourceNamespace = meta_v1.NamespaceAll
+			}
+			err := kubernetes.PrintComponents(componentsName, resourceNamespace, componentsOutputFormat)
 			if err != nil {
 				print.FailureStatusEvent(os.Stderr, err.Error())
 				os.Exit(1)
 			}
 		}
 	},
 	Example: `
-# List Kubernetes components
+# List Kubernetes Dapr components
```

Comment:
Mention that this will list only in default namespace,

Reply:
The NAMESPACE property for Dapr will not be affected by this change. This is changing the default behavior of the list, components and configurations command.